trusts/decorators.py

```diff
@@ -42,6 +108,7 @@ def _wrapped_view(request, *args, **kwargs):
         return _wrapped_view
     return decorator
 
+
```

Comment:
Okay. Follows PEP-8.

I didn't know the "standard" distingish between top level and not top level.